woocommerce/woocommerce-ios
# [Mobile Payments] Enable IPP UK Expansion Feature Flag

Closes: #9426


## Description

With this PR we enable the feature flag to the IPP UK Expansion feature, in preparation for its release. We leave the feature flag in place for now, in case we have to turn it off again quickly.
## Testing instructions


- Run the application with Debug Build Configuration, it should enable you to take payments with a UK-based store.
- Run the application with Release Build Configuration (see screenshot), it should enable you to take payments with a UK-based store.

## Screenshots

<img width="916" alt="Screenshot 2023-04-11 at 10 17 02" src="https://user-images.githubusercontent.com/1864060/231099360-5b95f789-c854-4114-9460-816c064a415c.png">



---
